examples/advanced/stochastic_odes.py

```diff
@@ -59,7 +59,7 @@ def exact_solution(t, dt):
 cols = ceil(1.0 * len(dts) / rows)
 errors = dict([(method, zeros(len(dts))) for method in methods])
 for dt_idx, dt in enumerate(dts):
-    print 'dt: ', dt
+    print('dt:', dt)
```

Comment:
Ah indeed, I forgot about this. What do you think would be the best solution?

1. Add `from __future__ import print_function` to each file
2. Use `'dt: ' + str(dt)`
3. Use `'dt: %s' % dt
4. Use `'dt: {0}'.format(str(dt))` (or a variant of this)

Some examples already use variants of 2 and 3, actually...